### ISSUE TITLE
⚡ Feature | Support embedding videos

### ISSUE THREAD

**Issue by friedger** [pradel/sigle#130]
## Description
Similar to images, it should be possible to upload or link videos

Here is a reference to a gaia hosted video stream: https://github.com/stacksgov/Stacks-Grants/issues/4#issuecomment-687810783

**Comment by pradel** [comment]
Agree, we should be able to link videos from different sources (youtube, vimeo and other alternatives..).
In a more general approach we should be able to include music from souncloud or other platforms.

**Comment by pradel** [comment]
TipTap added https://tiptap.dev/api/nodes/youtube but limited to Youtube

**Comment by friedger** [comment]
Source to upload videos to gaia in draft and public mode is here https://github.com/gaideo/gaideo

It might require a server that does the encoding.
